automation/run-tests.sh

```diff
@@ -299,6 +299,9 @@ if [[ -v customize_cmd ]];then
     container_exec "${customize_cmd}"
 fi
 
+container_exec "echo '/tmp/core.%h.%e.%t' > /proc/sys/kernel/core_pattern"
```

Comment:
How about specifying "CONT_EXPORT_DIR/" here directly instead of creating the cores first in /tmp and copying them afterwards?

Reply:
Sure. Will do.